CODEOWNERS

```diff
@@ -65,15 +65,67 @@ Dockerfile* @cilium/build
 /daemon/cmd/proxy.go @cilium/proxy
 /daemon/cmd/state.go @cilium/endpoint
 /daemon/cmd/sysctl_linux.go @cilium/bpf
+/Documentation/spelling_wordlist.txt @cilium/docs-structure
+/Documentation/community.rst @cilium/contributing
+/Documentation/check-crd-compat-table.sh @cilium/docs-structure
+/Documentation/glossary.rst @cilium/docs-structure
+/Documentation/commit-access.rst @cilium/contributing @cilium/docs-structure
+/Documentation/gettinghelp.rst @cilium/contributing @cilium/docs-structure
+/Documentation/gettingstarted/ipam.rst @cilium/ipam @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-azure.rst @cilium/azure @cilium/docs-structure
+/Documentation/gettingstarted/kubeproxy-free.rst @cilium/loadbalancer @cilium/docs-structure
+/Documentation/gettingstarted/docker.rst @cilium/docker @cilium/docs-structure
+/Documentation/gettingstarted/aws* @cilium/aws @cilium/docs-structure
+/Documentation/gettingstarted/cni-chaining-aws-cni.rst @cilium/aws @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-eks.rst @cilium/aws @cilium/docs-structure
+/Documentation/gettingstarted/bandwidth-manager.rst @cilium/bpf @cilium/docs-structure
+/Documentation/gettingstarted/index.rst @cilium/docs-structure
+/Documentation/gettingstarted/cni-chaining-azure-cni.rst @cilium/azure @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-azure-cni-validate.rst @cilium/azure @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-remove-aws-node.rst @cilium/aws @cilium/docs-structure
+/Documentation/gettingstarted/kind-configure.rst @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-azure-cni-steps.rst @cilium/azure @cilium/docs-structure
+/Documentation/gettingstarted/k8s-install-aks* @cilium/azure @cilium/docs-structure
+/Documentation/operations/performance/ @cilium/bpf @cilium/docs-structure
+/Documentation/policy/ @cilium/policy @cilium/docs-structure
+/Documentation/configuration/index.rst @cilium/docs-structure
```

Comment:
Isn't this already applied by matching `/Documentation/` for this group? (likewise for similar other paths which only have the @cilium/docs-structure group as owner, unless there may be a conflict with another line in the file)

Reply:
Yep, that's a good question. I wanted to keep the entries I've already classified as being @cilium/docs-structure's responsibility separate from entries I haven't classified under any codeowner because it's unclear (e.g., `Documentation/gettingstarted/local-redirect-policy.rst`).

I'm not sure that makes sense, especially considering it makes the whole thing more complex. But it's clearly easier to change the PR from separated entries to single @cilium/docs-structure entry than the other way around :smile: